### ISSUE TITLE
Preformatted block does not display preformatted text.

### ISSUE THREAD

**Issue by LuckyResistor** [WordPress/gutenberg#14188]
**Describe the bug**
The text in a preformatted block is not displayed preformatted. Newlines are not handled correctly.

**To Reproduce**
1. Create a block like:
```
<!-- wp:preformatted -->
<pre class="wp-block-preformatted">
Line 1
Line 2
Line 3
</pre>
<!-- /wp:preformatted -->
```
2. Switch to the visual editor.
3. The content is displayed like this:
![block](https://user-images.githubusercontent.com/9452552/53634444-7fa75600-3c1a-11e9-9300-a1a282df78cf.png)
- Newlines are not handled correctly for the block.
- The preview is correct:
![preview](https://user-images.githubusercontent.com/9452552/53634523-a8c7e680-3c1a-11e9-95fa-844af8d703fb.png)

**Expected behavior**
- Preformatted blocks shall be displayed correctly.

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Safari
 - Version: 12.0.3

**Additional context**
 - Gutenberg 5.1.1

**Comment by eidolonnight** [comment]
Confirmed in test site. gif of issue below:

![pretextest](https://user-images.githubusercontent.com/115154/53650581-ddd03b00-3c12-11e9-8269-b850f63c987a.gif)

Text does appear properly formatted in live post:

![image](https://user-images.githubusercontent.com/115154/53650635-f7718280-3c12-11e9-9de9-091a66884aea.png)

Workaround: Formatting appears to be permanently lost if, and only if, the Preformatted block is edited visually and then the page/post is saved. If this content is switched to "Edit as HTML", edited, and then saved, formatting is preserved.

**Comment by iseulde** [comment]
This was fixed in #13799.
